http/http-api/src/main/java/com/okta/commons/http/config/HttpClientConfiguration.java

```diff
@@ -141,6 +160,62 @@ public HttpClientConfiguration setRetryMaxAttempts(int retryMaxAttempts) {
         return this;
     }
 
+    public void setRequestExecutorParams(Map<String, String> map) {
+        if (map != null) {
+            for (Map.Entry<String, String> entry : map.entrySet()) {
+                this.requestExecutorParams.put(entry.getKey(), entry.getValue());
```

Comment:
use `putAll`
But this doesn't `set` the value, it's equivalent to `putAll`, a `set` should clear the previous values (or reassign the value)

Reply:
updated